wdn/templates_5.2/scss/critical.scss

```diff
@@ -26,9 +26,9 @@ html {
   font-style: normal;
   font-weight: 400;
 }
-h1 {
-  margin: .67em 0;
-}
+// h1 {
```

Comment:
Remove?

Reply:
The critical CSS is manually constructed from existing stylesheets. I commented rather than removed code so I could effectively toggle as many high-priority styles as possible until we got to ~13KB. Revisit in a separate issue?